Misc/NEWS.d/next/Build/2018-07-27-09-52-48.bpo-34245.bBV0NI.rst

```diff
@@ -0,0 +1,2 @@
+The Python library is now installed with write permission (mode 0755), which
+is the standard way of installing libraries.
```

Comment:
"library" is unclear word.  "py?" files are "Python library" too.
I think "shared module" or "DLL" are bettor wording.

Reply:
"Shared library" is probably the correct term. This change does not apply to the Windows DLLs or pyds, and using that term here may be confusing.